### PR TITLE
Error dispatching functions added

### PR DESCRIPTION
**Pull request by AtbTheGreat**

### CODE REVIEW

File: app/components/EditorPanel/Viewer.js [hunk 1]
```diff
@@ -27,9 +28,9 @@ export const Viewer = props => {
               })
             )
             .then(() => dispatch({ type: SET_EDIT_STATE }))
-            .catch(err =>
-              console.error("Error in exiting checkpoint view: ", err)
-            )
+            .catch(err => {
```
Comment (drewatk) [inline app/components/EditorPanel/Viewer.js]: Can you keep the `console.error`?

File: app/components/FileList/index.js [hunk 2]
```diff
@@ -19,9 +20,9 @@ const FileList = ({ files, gitAbstractions, currentFile, dispatch }) => {
               .createFile(fileName, convertToRaw(emptyFileContentState))
               .then(() => dispatch(updateFiles(gitAbstractions.getFiles())))
               .then(() => dispatch(selectFile(gitAbstractions, fileName)))
-              .catch(err =>
-                console.error("Error in CreateFileForm onSubmit: ", err)
-              );
+              .catch(err => {
```
Comment (drewatk) [inline app/components/FileList/index.js]: Keep the console.error please

File: app/containers/EditorPage.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const EditorPage = ({ showHistory, showFileList, currentFile }) => (
   <ErrorModal>
     <div>
       <TitleBar />
-      <div className={`${styles.container} conatiner-fluid`}>
+      <div className={styles.container}>
```
Comment (drewatk) [inline app/containers/EditorPage.js]: good catch 👍

File: app/components/FileList/index.js (hunk 2 above)
Comment (AtbTheGreat) [reply app/components/FileList/index.js]: Done & done

File: app/containers/EditorPage.js (hunk 3 above)
Comment (AtbTheGreat) [reply app/containers/EditorPage.js]: Thanks!

**Comment by marsha88** [comment]
Should we remove all the `console.error` calls or leave them for debugging purposes?

**Pull request merged**